staff/AbelusM/css-layout-simple-adaptive/styles/desktop.css

```diff
@@ -0,0 +1,50 @@
+
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+html, body {
+            min-height: 100% !important;
+            height: 100%;
+            color: white;
+            text-align: center;
+        }
+header {  
+    background-color:black; 
+    height: 20%; 
+    width: 100%; 
+    border: 1px solid white;
+}
+main{
+    border: 1px solid white;
+    height: 60%;
+    background-color: black;
+}
+nav{
+    border: 1px solid white;
+    background-color: black;
+    height: 100%;
+    width: 20%;
+    float: left;   
+}
+aside { 
+    border: 1px solid white;
+    background-color: black;
+    height: 100%;
+    width: 20%;
+    float: left;
+}   
+section {
+    border: 1px solid white;
+    background-color: black;
```

Comment:
TIP group selectors that share same rules (i.e., group shared rules in group of selectors)

SEE Grouping Selectors https://www.w3schools.com/css/css_syntax.asp

Reply:
thank you, done.